e2e/worker/test.sh

```diff
@@ -23,182 +47,6 @@ add_trap() {
 }
 
 
-message "# Case 0; try to build a target that will never succeed"
```

Comment:
these assertions all moved into e2e/test bats tests?

Reply:
is e2e/worker more of a smoke test now?

Reply:
everything inside this file is now in e2e/test bats files. i am not sure if i should remove this or not.

Reply:
probably good to have a WORKSPACE smoke test with worker mode if it isn't covered elsewhere

Reply:
I'll simply make it to be a smoke test and demonstrate different attributes of ts_project